File: Programming-languages-and-laboratory/Java/Exercises/29_compito_java_2019_04_11b/Guida.java
import java.util.*;

public class Guida extends Impiegato {

	private int app;
	private boolean inglese;
	private int tel;

	public Guida(int codice, String nome, int tel, boolean inglese, int app, double costo) {
		super(codice, nome, "guida", costo);
		this.app = app;
		this.tel = tel;
		this.inglese = inglese;
	}

	public String toString() {
		return super.toString() + "\t-\t-\t" + tel + "\t" + inglese + "\t" + app + "\t" + getCosto();
	}

}

File: Programming-languages-and-laboratory/Python/Exercises/29b_compito_java_20190411/Guida.py
from Impiegato import Impiegato


class Guida(Impiegato):
    def __init__(self, codice, tipo, nome, telefono, inglese, app_settimanali, costo):
        super().__init__(codice, tipo, nome, costo)
        self._app_settimanali = app_settimanali
        self._telefono = telefono
        self._inglese = inglese

    def __str__(self):
        return super().__str__() + "\tguida\t-\t-\t" + str(self._telefono) + "\t" + str(self._inglese) + "\t" + \
            str(self._app_settimanali) + "\t" + str(self._costo)
